### ISSUE TITLE
Redundant `yarn.lock` in `packages/web`

### ISSUE THREAD

**Issue by hanzili** [SwitchbackTech/compass#64]
## Prerequisites

- [x] Using an up-to-date main branch

## Expected Behavior
`yarn dev:web` should start the web development server without errors.

## Current Behavior
Running `yarn dev:web` results in the following error:
```
PS C:\Users\hanzi\Desktop\compass> yarn dev:web
Internal Error: @compass/web@workspace:.: This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile
...
```

## Steps to Reproduce

1. Clone the repository and switch to the main branch.
2. configure the .env
3. Run `yarn install at the root of the project.
5. Start the backend successfully.
6. Attempt to run `yarn dev:web `and encounter the above error.

## Possible Solution (Not obligatory)
The issue was resolved by removing a `yarn.lock` file found in the `packages/web` directory. After deleting this file and re-running `yarn install` at the root, `yarn dev:web` started working as expected. It seems that the presence of this additional `yarn.lock` file was causing conflicts with dependency resolution.

## Context
This issue may affect new contributors or setups, as the error prevents the web development server from starting. 
I am new to the project and wanted to confirm if this is an aspect that others have encountered or if it's a change worth considering for the project.

**Comment by tyler-dane** [comment]
Hey @hanzili, thanks for taking the time to debug and submit this issue! You're right, that `packages/web/yarn.lock` is redundant. Looks like it snuck into the codebase a while ago without me noticing. 

I wasn't able to replicate your error, but that could be because I have some global packages installed locally. 

I did remove `packages/web/yarn.lock` from a clean clone and got the web server running without errors, though, so I feel confident that removing it won't do extra harm.

So ... are you interested in submitting a PR that:
- Deletes that file
- Updates the `.gitignore` to ignore any nested `yarn.lock` files (while also allowing the root `yarn.lock`)

I'm chipping away at a few other issues currently, so getting your help on this would be great if you're willing.

**Comment by hanzili** [comment]
Thanks for your review! PR submited!

**Comment by tyler-dane** [comment]
Merged #67 , which closes this
